src/components/posts/Post.js

```diff
@@ -3,28 +3,30 @@ import { Link } from 'react-router-dom';
 import { Box, Stack } from '@mui/material';
 import React from 'react';
 import PostComments from './MainCommentComponent';
+import ProfileImg from '../ProfileImg/ProfileImg';
 
 function Post({ post, onCommentAdded }) {
-  return (
-    <>
-      {
-        <Box>
-          <div className='post-content'>{post.content}</div>
-          <Stack className='names-date' spacing={2} direction='row'>
-            <Link to={`/user/${post.user.id}`} className='post-author'>
-              <Box className='fullname' variant='contained'>
-                <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
-              </Box>
-            </Link>
-            <Box className='date-time' variant='contained'>
-              {dateTimetoRelativeTime(post.createdAt)}
-            </Box>
-          </Stack>
-        </Box>
-      }
-      <PostComments onCommentAdded={onCommentAdded} post={post} />
-    </>
-  );
+	return (
+		<>
+			{
```

Comment:
I don't think this curly bracket is needed here. You may be able to remove it, and save on some indentation!

Reply:
sorted